### ISSUE TITLE
APK doesn't work

### ISSUE THREAD

**Issue by Ind1eed** [Scirra/Construct-3-bugs#1022]
## Problem description

Android apk doesn't work, i got "error during extracting package"

## Attach a .c3p

Provided to the support.

## Steps to reproduce

1.
2.
3.

## Observed result

i see "error during extracting package"

## Expected result

Working apk.

## Affected browsers

* Chrome: yes

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by shortercode** [comment]
Please fill in some of the information, how did you build the APK. What settings did you use etc. I'm waiting for support to forward me the project so I literally have no information right now.

**Comment by shortercode** [comment]
I tried building your project and installing in on my test device, and it seemed to work fine. I'll need some more information on what you were doing.

**Comment by Ind1eed** [comment]
I'm just exporting the debug apk.
Also, but i guess it doesn't matter, i use light theme.


2017-11-20 15:17 GMT+01:00 Iain Shorter <notifications@github.com>:

**Comment by Ind1eed** [comment]
Im working in incognito mode cause kaspersky is breaking my projects.

2017-11-20 15:27 GMT+01:00 Piotr Piotr <npiotr96@gmail.com>:

**Comment by shortercode** [comment]
I don't seem to have any issue building your project and installing it, have you managed to install APK's from C3 before this? What version on Android is your device running? Incognito mode shouldn't have an effect.

**Comment by Ind1eed** [comment]
Yes, my other projects works perfectly exported via c3, too, My device is
Samsung Galaxy Note 4 (Android 6.0.1)

2017-11-20 15:38 GMT+01:00 Iain Shorter <notifications@github.com>:

**Comment by shortercode** [comment]
Just thought I'd point out that your background texture is too large to be rendered by my device, but that wouldn't prevent it from being installed. Try to stick to below 4096 x 4096.

Did you try building it again? Could be the file got corrupted during the transfer or similar

**Comment by Ind1eed** [comment]
You mean that i should resize the background texture to 4096x4096?

I tried once again, unchecked the script minification etc, still the same
error.

2017-11-20 15:55 GMT+01:00 Iain Shorter <notifications@github.com>:

**Comment by shortercode** [comment]
Well anything up to 4096 x 4096 really, it's up to you. You can probably manage a very similar quality at 2048 x 2048 and below. Incidentally most phone screens are 1920x1080, with some flagships being 2560x1440. I'm not aware of any phones that actually have enough pixels to show the level of detail your using.

Seems weird, can you try that with just a blank project for me? The runtime shouldn't make any difference to it really.

**Comment by Ind1eed** [comment]
The blank project loads properly.

20.11.2017 16:24 "Iain Shorter" <notifications@github.com> napisał(a):

**Comment by shortercode** [comment]
Does your device physically have the space to install the APK?

**Comment by Ind1eed** [comment]
Yes, it has enough space.

20.11.2017 16:40 "Iain Shorter" <notifications@github.com> napisał(a):

**Comment by shortercode** [comment]
could you try installing this [version](https://www.dropbox.com/s/w117q9tn9vygloo/SNOWMAN%20FACTORY.android.debug.apk?dl=0)?

**Comment by Ind1eed** [comment]
Doesn't work.

20.11.2017 16:47 "Iain Shorter" <notifications@github.com> napisał(a):

**Comment by shortercode** [comment]
Have you tried building it on the actual device and installing it there? Have to say I'm running out of ideas here, there's no reason I can see that it should be installing the APK. Unless something has modified the file in some way before it's installed.

**Comment by Ind1eed** [comment]
Yes i tried. It doesn't seems to work.

20.11.2017 5:06 PM "Iain Shorter" <notifications@github.com> napisał(a):

**Comment by shortercode** [comment]
I think I might have found something, testing at the moment. Try with a lowercase package ID

**Comment by shortercode** [comment]
Can confirm it fixed the issue on a Moto X2

**Comment by Ind1eed** [comment]
I changed the package ID to lowercase, and it seems to work fine :)
Thank you so much.

20.11.2017 5:22 PM "Iain Shorter" <notifications@github.com> napisał(a):

**Comment by shortercode** [comment]
I was worried that I wasn't going to find anything, seems like some devices require the package ID to be lowercase which is just weird.

Luckily one of the devices I tried happened to have this issue, and got a slightly more descriptive error ( INSTALL_PARSE_FAILED_MANIFEST_MALFORMED ) from Android Studio when I tried to install on it.

I guess this means the editor should at least try to encourage people to use lowercase package ID's, I expect forcing a lowercase package ID could cause some backwards compatibility issues so I want to avoid that!
